projects/kit/components/stepper/stepper.component.ts

```diff
@@ -71,9 +70,9 @@ export class TuiStepperComponent {
     }
 
     public indexOf(step: HTMLElement): number {
-        return tuiGetOriginalArrayFromQueryList(this.steps).findIndex(
```

Comment:
It looked like it could be replaced, correct if wrong

Reply:
It is used by `StepComponent` to determine if it is the active one and to activate it. `toArray` creates a new array which is a bit slower than `tuiGetOriginalArrayFromQueryList`. We can get rid of `indexOf` in favor of 
```ts
isActive(step: HTMLElement): boolean {
  return this.steps.get(this.activeItemIndex)?.nativeElement === step;
}
```
and we also need to change `activate` method to accept `HTMLElement` instead of index then.

Reply:
Changing `activate` method looks like a breaking change. This is a public method, perhaps someone uses it in their projects. Is this really necessary? Maybe just return `tuiGetOriginalArrayFromQueryList`

Reply:
Sure, let's just return it.